<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>S5 Allele-Specific ChIP-seq Analyses | Supplementary Materials</title>
  <meta name="description" content="S5 Allele-Specific ChIP-seq Analyses | Supplementary Materials" />
  <meta name="generator" content="bookdown 0.38 and GitBook 2.6.7" />

  <meta property="og:title" content="S5 Allele-Specific ChIP-seq Analyses | Supplementary Materials" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="S5 Allele-Specific ChIP-seq Analyses | Supplementary Materials" />
  
  
  

<meta name="author" content="Sean T. Bresnahan, Shaun Mahony, Kate Anton, Brock Harpur, Christina M. Grozinger" />


<meta name="date" content="2024-05-07" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="chip-seq-analysis.html"/>
<link rel="next" href="hptm-transcription-dynamics.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Appendix D</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a></li>
<li class="chapter" data-level="1" data-path="wgs-mrna-seq-and-chip-seq.html"><a href="wgs-mrna-seq-and-chip-seq.html"><i class="fa fa-check"></i><b>1</b> WGS, mRNA-seq, and ChIP-seq</a>
<ul>
<li class="chapter" data-level="1.1" data-path="wgs-mrna-seq-and-chip-seq.html"><a href="wgs-mrna-seq-and-chip-seq.html#wgs"><i class="fa fa-check"></i><b>1.1</b> WGS</a></li>
<li class="chapter" data-level="1.2" data-path="wgs-mrna-seq-and-chip-seq.html"><a href="wgs-mrna-seq-and-chip-seq.html#mrna-seq"><i class="fa fa-check"></i><b>1.2</b> mRNA-seq</a></li>
<li class="chapter" data-level="1.3" data-path="wgs-mrna-seq-and-chip-seq.html"><a href="wgs-mrna-seq-and-chip-seq.html#chip-seq"><i class="fa fa-check"></i><b>1.3</b> ChIP-seq</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="transcriptomic-analyses.html"><a href="transcriptomic-analyses.html"><i class="fa fa-check"></i><b>2</b> Transcriptomic Analyses</a>
<ul>
<li class="chapter" data-level="2.1" data-path="transcriptomic-analyses.html"><a href="transcriptomic-analyses.html#differential-gene-expression-analysis"><i class="fa fa-check"></i><b>2.1</b> Differential gene expression analysis</a></li>
<li class="chapter" data-level="2.2" data-path="transcriptomic-analyses.html"><a href="transcriptomic-analyses.html#wgcna"><i class="fa fa-check"></i><b>2.2</b> WGCNA</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html"><i class="fa fa-check"></i><b>3</b> Allele-Specific Transcriptomic Analyses</a>
<ul>
<li class="chapter" data-level="3.1" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#snps-and-transcripts"><i class="fa fa-check"></i><b>3.1</b> SNPs and transcripts</a></li>
<li class="chapter" data-level="3.2" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#poe-gene-lists"><i class="fa fa-check"></i><b>3.2</b> POE gene lists</a></li>
<li class="chapter" data-level="3.3" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#hpf-wl-ql---allelic-transcriptomics"><i class="fa fa-check"></i><b>3.3</b> 192hpf WL &amp; QL - allelic transcriptomics</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#block-1"><i class="fa fa-check"></i><b>3.3.1</b> Block 1</a></li>
<li class="chapter" data-level="3.3.2" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#block-2"><i class="fa fa-check"></i><b>3.3.2</b> Block 2</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#hpf-eggs---allelic-transcriptomics"><i class="fa fa-check"></i><b>3.4</b> 24hpf eggs - allelic transcriptomics</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#block-3"><i class="fa fa-check"></i><b>3.4.1</b> Block 3</a></li>
<li class="chapter" data-level="3.4.2" data-path="allele-specific-transcriptomic-analyses.html"><a href="allele-specific-transcriptomic-analyses.html#block-4"><i class="fa fa-check"></i><b>3.4.2</b> Block 4</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="chip-seq-analysis.html"><a href="chip-seq-analysis.html"><i class="fa fa-check"></i><b>4</b> ChIP-seq Analysis</a>
<ul>
<li class="chapter" data-level="4.1" data-path="chip-seq-analysis.html"><a href="chip-seq-analysis.html#enrichment-across-promoters-and-gene-bodies"><i class="fa fa-check"></i><b>4.1</b> Enrichment across promoters and gene bodies</a></li>
<li class="chapter" data-level="4.2" data-path="chip-seq-analysis.html"><a href="chip-seq-analysis.html#differential-enrichment-analysis"><i class="fa fa-check"></i><b>4.2</b> Differential enrichment analysis</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="chip-seq-analysis.html"><a href="chip-seq-analysis.html#h3k27me3"><i class="fa fa-check"></i><b>4.2.1</b> H3K27me3</a></li>
<li class="chapter" data-level="4.2.2" data-path="chip-seq-analysis.html"><a href="chip-seq-analysis.html#h3k27ac"><i class="fa fa-check"></i><b>4.2.2</b> H3K27ac</a></li>
<li class="chapter" data-level="4.2.3" data-path="chip-seq-analysis.html"><a href="chip-seq-analysis.html#h3k4me3"><i class="fa fa-check"></i><b>4.2.3</b> H3K4me3</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html"><i class="fa fa-check"></i><b>5</b> Allele-Specific ChIP-seq Analyses</a>
<ul>
<li class="chapter" data-level="5.1" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#snps-and-peaks"><i class="fa fa-check"></i><b>5.1</b> SNPs and peaks</a></li>
<li class="chapter" data-level="5.2" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#poe-peak-lists"><i class="fa fa-check"></i><b>5.2</b> POE peak lists</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#h3k27me3-1"><i class="fa fa-check"></i><b>5.2.1</b> H3K27me3</a></li>
<li class="chapter" data-level="5.2.2" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#h3k27ac-1"><i class="fa fa-check"></i><b>5.2.2</b> H3K27ac</a></li>
<li class="chapter" data-level="5.2.3" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#h3k4me3-1"><i class="fa fa-check"></i><b>5.2.3</b> H3K4me3</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#h3k27me3-2"><i class="fa fa-check"></i><b>5.3</b> H3K27me3</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#gene-bodies"><i class="fa fa-check"></i><b>5.3.1</b> Gene bodies</a></li>
<li class="chapter" data-level="5.3.2" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#promoters"><i class="fa fa-check"></i><b>5.3.2</b> Promoters</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#h3k27ac-2"><i class="fa fa-check"></i><b>5.4</b> H3K27ac</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#gene-bodies-1"><i class="fa fa-check"></i><b>5.4.1</b> Gene bodies</a></li>
<li class="chapter" data-level="5.4.2" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#promoters-1"><i class="fa fa-check"></i><b>5.4.2</b> Promoters</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#h3k4me3-2"><i class="fa fa-check"></i><b>5.5</b> H3K4me3</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#gene-bodies-2"><i class="fa fa-check"></i><b>5.5.1</b> Gene bodies</a></li>
<li class="chapter" data-level="5.5.2" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#promoters-2"><i class="fa fa-check"></i><b>5.5.2</b> Promoters</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#combined-results"><i class="fa fa-check"></i><b>5.6</b> Combined results</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#block-1-7"><i class="fa fa-check"></i><b>5.6.1</b> Block 1</a></li>
<li class="chapter" data-level="5.6.2" data-path="allele-specific-chip-seq-analyses.html"><a href="allele-specific-chip-seq-analyses.html#block-2-7"><i class="fa fa-check"></i><b>5.6.2</b> Block 2</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html"><i class="fa fa-check"></i><b>6</b> HPTM-transcription dynamics</a>
<ul>
<li class="chapter" data-level="6.1" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#h3k27me3-3"><i class="fa fa-check"></i><b>6.1</b> H3K27me3</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#gene-bodies-3"><i class="fa fa-check"></i><b>6.1.1</b> Gene bodies</a></li>
<li class="chapter" data-level="6.1.2" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#promoters-3"><i class="fa fa-check"></i><b>6.1.2</b> Promoters</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#h3k27ac-3"><i class="fa fa-check"></i><b>6.2</b> H3K27ac</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#gene-bodies-4"><i class="fa fa-check"></i><b>6.2.1</b> Gene bodies</a></li>
<li class="chapter" data-level="6.2.2" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#promoters-4"><i class="fa fa-check"></i><b>6.2.2</b> Promoters</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#h3k4me3-3"><i class="fa fa-check"></i><b>6.3</b> H3K4me3</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#gene-bodies-5"><i class="fa fa-check"></i><b>6.3.1</b> Gene bodies</a></li>
<li class="chapter" data-level="6.3.2" data-path="hptm-transcription-dynamics.html"><a href="hptm-transcription-dynamics.html#promoters-5"><i class="fa fa-check"></i><b>6.3.2</b> Promoters</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html"><i class="fa fa-check"></i><b>7</b> POE HPTM enrichment-POE transcription dynamics</a>
<ul>
<li class="chapter" data-level="7.1" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#block-1-8"><i class="fa fa-check"></i><b>7.1</b> Block 1</a>
<ul>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27me3-4"><i class="fa fa-check"></i>H3K27me3</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27ac-4"><i class="fa fa-check"></i>H3K27ac</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k4me3-4"><i class="fa fa-check"></i>H3K4me3</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#block-2-8"><i class="fa fa-check"></i><b>7.2</b> Block 2</a>
<ul>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27me3-5"><i class="fa fa-check"></i>H3K27me3</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27ac-5"><i class="fa fa-check"></i>H3K27ac</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k4me3-5"><i class="fa fa-check"></i>H3K4me3</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#logistic-regression"><i class="fa fa-check"></i><b>7.3</b> Logistic regression</a>
<ul>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27me3-gene-body"><i class="fa fa-check"></i>H3K27me3 (gene body)</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27me3-promoter"><i class="fa fa-check"></i>H3K27me3 (promoter)</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27ac-gene-body"><i class="fa fa-check"></i>H3K27ac (gene body)</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k27ac-promoter"><i class="fa fa-check"></i>H3K27ac (promoter)</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k4me3-gene-body"><i class="fa fa-check"></i>H3K4me3 (gene body)</a></li>
<li class="chapter" data-level="" data-path="poe-hptm-enrichment-poe-transcription-dynamics.html"><a href="poe-hptm-enrichment-poe-transcription-dynamics.html#h3k4me3-promoter"><i class="fa fa-check"></i>H3K4me3 (promoter)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="go-enrichment-analyses.html"><a href="go-enrichment-analyses.html"><i class="fa fa-check"></i><b>8</b> GO enrichment analyses</a>
<ul>
<li class="chapter" data-level="8.1" data-path="go-enrichment-analyses.html"><a href="go-enrichment-analyses.html#degs-poe-genes"><i class="fa fa-check"></i><b>8.1</b> DEGs &amp; POE genes</a></li>
<li class="chapter" data-level="8.2" data-path="go-enrichment-analyses.html"><a href="go-enrichment-analyses.html#wgcna-modules"><i class="fa fa-check"></i><b>8.2</b> WGCNA modules</a></li>
<li class="chapter" data-level="8.3" data-path="go-enrichment-analyses.html"><a href="go-enrichment-analyses.html#hptm-differentially-enriched-regions"><i class="fa fa-check"></i><b>8.3</b> HPTM differentially enriched regions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="poe-gene-feature-assessment.html"><a href="poe-gene-feature-assessment.html"><i class="fa fa-check"></i><b>9</b> POE gene feature assessment</a>
<ul>
<li class="chapter" data-level="9.1" data-path="poe-gene-feature-assessment.html"><a href="poe-gene-feature-assessment.html#spatial-clustering"><i class="fa fa-check"></i><b>9.1</b> Spatial clustering</a></li>
<li class="chapter" data-level="9.2" data-path="poe-gene-feature-assessment.html"><a href="poe-gene-feature-assessment.html#intron-feature-analyses"><i class="fa fa-check"></i><b>9.2</b> Intron feature analyses</a>
<ul>
<li class="chapter" data-level="" data-path="poe-gene-feature-assessment.html"><a href="poe-gene-feature-assessment.html#recombination-rate"><i class="fa fa-check"></i>Recombination rate</a></li>
<li class="chapter" data-level="" data-path="poe-gene-feature-assessment.html"><a href="poe-gene-feature-assessment.html#intron"><i class="fa fa-check"></i>% intron</a></li>
<li class="chapter" data-level="" data-path="poe-gene-feature-assessment.html"><a href="poe-gene-feature-assessment.html#intron-gc"><i class="fa fa-check"></i>Intron GC</a></li>
<li class="chapter" data-level="" data-path="poe-gene-feature-assessment.html"><a href="poe-gene-feature-assessment.html#intron-cpg-density"><i class="fa fa-check"></i>Intron CpG density</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supplementary Materials</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="allele-specific-chip-seq-analyses" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">S5</span> Allele-Specific ChIP-seq Analyses<a href="allele-specific-chip-seq-analyses.html#allele-specific-chip-seq-analyses" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="snps-and-peaks" class="section level2 hasAnchor" number="5.1">
<h2><span class="header-section-number">5.1</span> SNPs and peaks<a href="allele-specific-chip-seq-analyses.html#snps-and-peaks" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
HPTM
</th>
<th style="text-align:right;">
Block
</th>
<th style="text-align:left;">
Caste
</th>
<th style="text-align:right;">
SNPs
</th>
<th style="text-align:right;">
Peaks
</th>
<th style="text-align:right;">
SNPs per Peak
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
H3K27me3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
WL
</td>
<td style="text-align:right;">
79473
</td>
<td style="text-align:right;">
14160
</td>
<td style="text-align:right;">
5.61
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K27me3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
QL
</td>
<td style="text-align:right;">
82753
</td>
<td style="text-align:right;">
19814
</td>
<td style="text-align:right;">
4.18
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K27me3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
WL
</td>
<td style="text-align:right;">
61556
</td>
<td style="text-align:right;">
10919
</td>
<td style="text-align:right;">
5.64
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K27me3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
QL
</td>
<td style="text-align:right;">
73681
</td>
<td style="text-align:right;">
12119
</td>
<td style="text-align:right;">
6.08
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K27ac
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
WL
</td>
<td style="text-align:right;">
85908
</td>
<td style="text-align:right;">
15148
</td>
<td style="text-align:right;">
5.67
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K27ac
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
QL
</td>
<td style="text-align:right;">
90950
</td>
<td style="text-align:right;">
19701
</td>
<td style="text-align:right;">
4.62
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K27ac
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
WL
</td>
<td style="text-align:right;">
27763
</td>
<td style="text-align:right;">
6140
</td>
<td style="text-align:right;">
4.52
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K27ac
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
QL
</td>
<td style="text-align:right;">
55079
</td>
<td style="text-align:right;">
12344
</td>
<td style="text-align:right;">
4.46
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K4me3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
WL
</td>
<td style="text-align:right;">
66288
</td>
<td style="text-align:right;">
14356
</td>
<td style="text-align:right;">
4.62
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K4me3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
QL
</td>
<td style="text-align:right;">
65804
</td>
<td style="text-align:right;">
17152
</td>
<td style="text-align:right;">
3.84
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K4me3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
WL
</td>
<td style="text-align:right;">
51650
</td>
<td style="text-align:right;">
10242
</td>
<td style="text-align:right;">
5.04
</td>
</tr>
<tr>
<td style="text-align:left;">
H3K4me3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
QL
</td>
<td style="text-align:right;">
58502
</td>
<td style="text-align:right;">
13038
</td>
<td style="text-align:right;">
4.49
</td>
</tr>
</tbody>
</table>
<ul>
<li><a href="resources/data/consensus_peaks.tar.gz">Consensus peak sets in BED format</a></li>
</ul>
</div>
<div id="poe-peak-lists" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> POE peak lists<a href="allele-specific-chip-seq-analyses.html#poe-peak-lists" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="h3k27me3-1" class="section level3 hasAnchor" number="5.2.1">
<h3><span class="header-section-number">5.2.1</span> H3K27me3<a href="allele-specific-chip-seq-analyses.html#h3k27me3-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li><a href="resources/data/block1_H3K27me3_GB_PSpeaks.csv">Block 1 - gene bodies</a></li>
<li><a href="resources/data/block1_H3K27me3_P_PSpeaks.csv">Block 1 - promoters</a></li>
<li><a href="resources/data/block2_H3K27me3_GB_PSpeaks.csv">Block 2 - gene bodies</a></li>
<li><a href="resources/data/block2_H3K27me3_P_PSpeaks.csv">Block 2 - promoters</a></li>
</ul>
</div>
<div id="h3k27ac-1" class="section level3 hasAnchor" number="5.2.2">
<h3><span class="header-section-number">5.2.2</span> H3K27ac<a href="allele-specific-chip-seq-analyses.html#h3k27ac-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li><a href="resources/data/block1_H3K27ac_GB_PSpeaks.csv">Block 1 - gene bodies</a></li>
<li><a href="resources/data/block1_H3K27ac_P_PSpeaks.csv">Block 1 - promoters</a></li>
<li><a href="resources/data/block2_H3K27ac_GB_PSpeaks.csv">Block 2 - gene bodies</a></li>
<li><a href="resources/data/block2_H3K27ac_P_PSpeaks.csv">Block 2 - promoters</a></li>
</ul>
</div>
<div id="h3k4me3-1" class="section level3 hasAnchor" number="5.2.3">
<h3><span class="header-section-number">5.2.3</span> H3K4me3<a href="allele-specific-chip-seq-analyses.html#h3k4me3-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li><a href="resources/data/block1_H3K4me3_GB_PSpeaks.csv">Block 1 - gene bodies</a></li>
<li><a href="resources/data/block1_H3K4me3_P_PSpeaks.csv">Block 1 - promoters</a></li>
<li><a href="resources/data/block2_H3K4me3_GB_PSpeaks.csv">Block 2 - gene bodies</a></li>
<li><a href="resources/data/block2_H3K4me3_P_PSpeaks.csv">Block 2 - promoters</a></li>
</ul>
</div>
</div>
<div id="h3k27me3-2" class="section level2 hasAnchor" number="5.3">
<h2><span class="header-section-number">5.3</span> H3K27me3<a href="allele-specific-chip-seq-analyses.html#h3k27me3-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Caste differentiation is associated with variation in POEs on H3K27me3 enrichment. Allele-specific enrichment at H3K27me3 peaks were assessed in F2 worker-destined larvae (WL) and queen-destined larvae (QL) at 192hpf collected from a reciprocal cross between different stocks of European honey bees. The x-axis represents, for each peak, the proportion of lineage A reads in bees with a lineage B mother and lineage A father. The y-axis represents, for each peak, the proportion of lineage A reads in bees with a lineage A mother and lineage B father (p2). Each color represents a peak which is significantly biased at all tested SNP positions. Significance was determined using the overlap between two statistical tests: a generalized linear interactive mixed model (GLIMMIX), and a Storer-Kim binomial exact test along with previously established thresholds of p1&lt;0.4 and p2&gt;0.6 for maternal bias, p1&gt;0.6 and p2&lt;0.4 for paternal bias, p1&lt;0.4 and p2&lt;0.4 for lineage B bias and p1&gt;0.6 and p2&gt;0.6 for lineage A bias.</p>
<div id="gene-bodies" class="section level3 hasAnchor" number="5.3.1">
<h3><span class="header-section-number">5.3.1</span> Gene bodies<a href="allele-specific-chip-seq-analyses.html#gene-bodies" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="block-1-1" class="section level4 hasAnchor" number="5.3.1.1">
<h4><span class="header-section-number">5.3.1.1</span> Block 1<a href="allele-specific-chip-seq-analyses.html#block-1-1" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block1_H3K27me3_GB.png" /></p>
</div>
<div id="block-2-1" class="section level4 hasAnchor" number="5.3.1.2">
<h4><span class="header-section-number">5.3.1.2</span> Block 2<a href="allele-specific-chip-seq-analyses.html#block-2-1" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block2_H3K27me3_GB.png" /></p>
</div>
</div>
<div id="promoters" class="section level3 hasAnchor" number="5.3.2">
<h3><span class="header-section-number">5.3.2</span> Promoters<a href="allele-specific-chip-seq-analyses.html#promoters" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="block-1-2" class="section level4 hasAnchor" number="5.3.2.1">
<h4><span class="header-section-number">5.3.2.1</span> Block 1<a href="allele-specific-chip-seq-analyses.html#block-1-2" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block1_H3K27me3_P.png" /></p>
</div>
<div id="block-2-2" class="section level4 hasAnchor" number="5.3.2.2">
<h4><span class="header-section-number">5.3.2.2</span> Block 2<a href="allele-specific-chip-seq-analyses.html#block-2-2" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block2_H3K27me3_P.png" /></p>
</div>
</div>
</div>
<div id="h3k27ac-2" class="section level2 hasAnchor" number="5.4">
<h2><span class="header-section-number">5.4</span> H3K27ac<a href="allele-specific-chip-seq-analyses.html#h3k27ac-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Caste differentiation is associated with variation in POEs on H3K27ac enrichment. Allele-specific enrichment at H3K27ac peaks were assessed in F2 worker-destined larvae (WL) and queen-destined larvae (QL) at 192hpf collected from a reciprocal cross between different stocks of European honey bees. The x-axis represents, for each peak, the proportion of lineage A reads in bees with a lineage B mother and lineage A father. The y-axis represents, for each peak, the proportion of lineage A reads in bees with a lineage A mother and lineage B father (p2). Each color represents a peak which is significantly biased at all tested SNP positions. Significance was determined using the overlap between two statistical tests: a generalized linear interactive mixed model (GLIMMIX), and a Storer-Kim binomial exact test along with previously established thresholds of p1&lt;0.4 and p2&gt;0.6 for maternal bias, p1&gt;0.6 and p2&lt;0.4 for paternal bias, p1&lt;0.4 and p2&lt;0.4 for lineage B bias and p1&gt;0.6 and p2&gt;0.6 for lineage A bias.</p>
<div id="gene-bodies-1" class="section level3 hasAnchor" number="5.4.1">
<h3><span class="header-section-number">5.4.1</span> Gene bodies<a href="allele-specific-chip-seq-analyses.html#gene-bodies-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="block-1-3" class="section level4 hasAnchor" number="5.4.1.1">
<h4><span class="header-section-number">5.4.1.1</span> Block 1<a href="allele-specific-chip-seq-analyses.html#block-1-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block1_H3K27ac_GB.png" /></p>
</div>
<div id="block-2-3" class="section level4 hasAnchor" number="5.4.1.2">
<h4><span class="header-section-number">5.4.1.2</span> Block 2<a href="allele-specific-chip-seq-analyses.html#block-2-3" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block2_H3K27ac_GB.png" /></p>
</div>
</div>
<div id="promoters-1" class="section level3 hasAnchor" number="5.4.2">
<h3><span class="header-section-number">5.4.2</span> Promoters<a href="allele-specific-chip-seq-analyses.html#promoters-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="block-1-4" class="section level4 hasAnchor" number="5.4.2.1">
<h4><span class="header-section-number">5.4.2.1</span> Block 1<a href="allele-specific-chip-seq-analyses.html#block-1-4" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block1_H3K27ac_P.png" /></p>
</div>
<div id="block-2-4" class="section level4 hasAnchor" number="5.4.2.2">
<h4><span class="header-section-number">5.4.2.2</span> Block 2<a href="allele-specific-chip-seq-analyses.html#block-2-4" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block2_H3K27ac_P.png" /></p>
</div>
</div>
</div>
<div id="h3k4me3-2" class="section level2 hasAnchor" number="5.5">
<h2><span class="header-section-number">5.5</span> H3K4me3<a href="allele-specific-chip-seq-analyses.html#h3k4me3-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Caste differentiation is associated with variation in POEs on H3K4me3 enrichment. Allele-specific enrichment at H3K4me3 peaks were assessed in F2 worker-destined larvae (WL) and queen-destined larvae (QL) at 192hpf collected from a reciprocal cross between different stocks of European honey bees. The x-axis represents, for each peak, the proportion of lineage A reads in bees with a lineage B mother and lineage A father. The y-axis represents, for each peak, the proportion of lineage A reads in bees with a lineage A mother and lineage B father (p2). Each color represents a peak which is significantly biased at all tested SNP positions. Significance was determined using the overlap between two statistical tests: a generalized linear interactive mixed model (GLIMMIX), and a Storer-Kim binomial exact test along with previously established thresholds of p1&lt;0.4 and p2&gt;0.6 for maternal bias, p1&gt;0.6 and p2&lt;0.4 for paternal bias, p1&lt;0.4 and p2&lt;0.4 for lineage B bias and p1&gt;0.6 and p2&gt;0.6 for lineage A bias.</p>
<div id="gene-bodies-2" class="section level3 hasAnchor" number="5.5.1">
<h3><span class="header-section-number">5.5.1</span> Gene bodies<a href="allele-specific-chip-seq-analyses.html#gene-bodies-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="block-1-5" class="section level4 hasAnchor" number="5.5.1.1">
<h4><span class="header-section-number">5.5.1.1</span> Block 1<a href="allele-specific-chip-seq-analyses.html#block-1-5" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block1_H3K4me3_GB.png" /></p>
</div>
<div id="block-2-5" class="section level4 hasAnchor" number="5.5.1.2">
<h4><span class="header-section-number">5.5.1.2</span> Block 2<a href="allele-specific-chip-seq-analyses.html#block-2-5" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block2_H3K4me3_GB.png" /></p>
</div>
</div>
<div id="promoters-2" class="section level3 hasAnchor" number="5.5.2">
<h3><span class="header-section-number">5.5.2</span> Promoters<a href="allele-specific-chip-seq-analyses.html#promoters-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="block-1-6" class="section level4 hasAnchor" number="5.5.2.1">
<h4><span class="header-section-number">5.5.2.1</span> Block 1<a href="allele-specific-chip-seq-analyses.html#block-1-6" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block1_H3K4me3_P.png" /></p>
</div>
<div id="block-2-6" class="section level4 hasAnchor" number="5.5.2.2">
<h4><span class="header-section-number">5.5.2.2</span> Block 2<a href="allele-specific-chip-seq-analyses.html#block-2-6" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="resources/figures/block2_H3K4me3_P.png" /></p>
</div>
</div>
</div>
<div id="combined-results" class="section level2 hasAnchor" number="5.6">
<h2><span class="header-section-number">5.6</span> Combined results<a href="allele-specific-chip-seq-analyses.html#combined-results" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Differences in allelic enrichment of HPTMs across promoters (P) and gene bodies (GB) between queen-destined larvae (QL) and worker-destined larvae (WL) at 192hpf in block 1. The log2 count of peaks showing significant maternal (mat), lineage A, lineage B, or paternal (pat) allele-specific enrichment across (a-c) promoters and (d-f) gene bodies at all tested SNP positions. In each panel, p-values for Chi-squared tests of independence for comparisons between the castes are indicated (NS = not significant, *<em>p</em>&lt;0.01, **<em>p</em>&lt;0.001, ***<em>p</em>&lt;0.001). Significance of allele-biased HPTM enrichment was determined using the overlap between two statistical tests: a general linear mixed model (GLIMMIX), and a Storer-Kim binomial exact test along with previously established thresholds of p1&lt;0.4 and p2&gt;0.6 for maternal bias, p1&gt;0.6 and p2&lt;0.4 for paternal bias, p1&lt;0.4 and p2&lt;0.4 for lineage B bias and p1&gt;0.6 and p2&gt;0.6 for lineage A bias.</p>
<div id="block-1-7" class="section level3 hasAnchor" number="5.6.1">
<h3><span class="header-section-number">5.6.1</span> Block 1<a href="allele-specific-chip-seq-analyses.html#block-1-7" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><img src="resources/figures/MSfig6.png" /></p>
</div>
<div id="block-2-7" class="section level3 hasAnchor" number="5.6.2">
<h3><span class="header-section-number">5.6.2</span> Block 2<a href="allele-specific-chip-seq-analyses.html#block-2-7" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><img src="resources/figures/MSfig6b.png" /></p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="chip-seq-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="hptm-transcription-dynamics.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/YOUR GITHUB USERNAME/YOUR REPO NAME/edit/main/05-allelic-chip.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/YOUR GITHUB USERNAME/YOUR REPO NAME/blob/main/05-allelic-chip.Rmd",
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
